apps/web/src/views/CartView.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import CartTable from '../../../../packages/ui/cart/CartTable';
+export const CartView = () => {
+  return (
+    <main>
+      {/* Section height is for demo purposes only; please position the footer at the bottom instead of the middle */}
+      <section style={{ height: '100vh', background: 'lightgray' }}>
```

Comment:
Use percentage than vh (viewheight)
```suggestion
      <section style={{ height: '100%', background: 'lightgray' }}>
```